lib/ui/window.dart

```diff
@@ -138,8 +138,42 @@ class FlutterView {
   ///    The value here is equal to the value exposed on [display].
   double get devicePixelRatio => _viewConfiguration.devicePixelRatio;
 
-  /// The dimensions of the rectangle into which the scene rendered in this view
-  /// will be drawn on the screen, in physical pixels.
+  /// The sizing constraints in physical pixels for this view.
+  ///
+  /// The view can take on any [Size] that fulfills these constraints. These
+  /// constraints are typically used by an UI framework as the input for its
+  /// layout algorithm to determine an approrpiate size for the view. To size
+  /// the view, the selected size must be provided to the [render] method and it
+  /// must satisfy the constraints.
+  ///
+  /// When this changes, [PlatformDispatcher.onMetricsChanged] is called.
```

Comment:
> `/// When this changes, [PlatformDispatcher...] is called.`

This bit of the documentation states that this may change, but there's not a setter for `physicalConstraints`. Should these be re-configurable after view creation? Do we need a new native API to `setPhysicalConstraints`?

Reply:
This is up to the embedders. If they want, they can offer an API to change this after View creation. If they do, `PlatformDispatcher.onMetricsChanged` needs to be invoked when it changes.

Reply:
Fantastic!